FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_ARP.c

```diff
@@ -302,6 +311,10 @@ uint8_t ucMinAgeFound = 0U;
 				ucMinAgeFound = xARPCache[ x ].ucAge;
 				xUseEntry = x;
 			}
+			else    /* Added to suppress MISRA rule 15.7 violation */
```

Comment:
You are not _suppressing_ a rule here, you are _conforming_ to it.  Delete the comment after the `else`.